auto_round/utils.py

```diff
@@ -980,3 +980,64 @@ def compile_func(fun, device, enable_torch_compile):
         return compile_func_on_cuda_or_cpu(fun, enable_torch_compile)
 
 
+
+def is_numba_available():  # pragma: no cover
+    """Check if Numba is available."""
+    try:
+        import numba
+
+        return True
+    except ImportError:
+        return False
+
+
+def _is_tbb_installed():  # pragma: no cover
+    import importlib.metadata
+
+    try:
+        importlib.metadata.version("tbb")
```

Comment:
better add a function, check_package_install or similar.  you may implement it via from transformers.utils.versions import require_version

Reply:
Agree, will port such helper functions in a separate PR.